assignments/arrays.js

```diff
@@ -63,32 +63,52 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+const car33 = inventory[33];
+console.log(`Car 33 is a ${car33.car_year} ${car33.car_make} ${car33.car_model}` );
```

Comment:
remember array index's start at 0, so to get car 33 you actually need to set it to inventory[33-1]

Reply:
@NickStrick I actually called the index 32 originally, but then I notice the word "zeroth", and I was like "are they trying to trick me" and change it to 33